doc/ref_model/chapters/chapter07.rst

```diff
@@ -1105,9 +1104,8 @@ System hardening
 | req.sec.gen.002 | The systems part of the cloud infrastructure     | Hardening: CIS Password Policy Guide            |
```

Comment:
This should look like this: 
![image](https://github.com/user-attachments/assets/6fc7ea50-0fc7-459c-a2c6-bc4d10cb66fe)

Reply:
Solved